legal-api/src/legal_api/services/authz.py

```diff
@@ -118,6 +118,17 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
             'adminFreeze': {
                 'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC'],
             },
+            'agmLocationChange': {
+                'legalTypes': ['BC', 'BEN', 'ULC', 'CC'],
+                'blockerChecks': {
+                    'business': [
+                        BusinessBlocker.DEFAULT,
+                        BusinessBlocker.BUSINESS_FROZEN,
+                        BusinessBlocker.DRAFT_PENDING,
```

Comment:
business frozen and draft pending is already contained in default so we can remove these two.